### PR TITLE
[DOCUMENTATION]: Developer Quickstart

### LINKED ISSUE #1: Determine the Specs of our Website
What infrastructure do we want to use on this project?

Backend:
- [x] Python? (Flask)

Frontend:
- [x] HTML
- [ ] CSS

### PR DESCRIPTION
**Pull request by BennyWestsyde**
I have written a developer quickstart for everyone to reference so that everyone can get up to speed with the workflow. Everyone has been requested to review this. please approve and comment on it once you have read it through. Additionally, if there are any more questions or things that we feel should be added please let me know.

This will close #1

@UTD-4347-Databosses/everyone

### CODE REVIEW

**Review by Tank52 — COMMENTED** [review]
Looks good. Suggest everyone takes a look, especially if they haven't worked with python before

File: docs/developer_quickstart.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+├── .gitignore
+├── .flaskenv
+├── requirements.txt <- pip can install these
+├── run.py <- Entry point
+```
+
+## Setting up the project
+1. Install necessary languages/tools:
+    - Python 3.8 or higher
+    - Git
```
Comment (BeTheChange07) [inline docs/developer_quickstart.md:31]: need command line associated with your linux version

File: docs/developer_quickstart.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+├── venv
+├── .gitignore
+├── .flaskenv
+├── requirements.txt <- pip can install these
+├── run.py <- Entry point
+```
+
+## Setting up the project
+1. Install necessary languages/tools:
+    - Python 3.8 or higher
```
Comment (BeTheChange07) [inline docs/developer_quickstart.md:30]: need terminal command associated with your linux version

**Review by BeTheChange07 — APPROVED** [review]
using it a lot

**Comment by BeTheChange07** [comment]
why is the html in this pull request?

**Review by BeTheChange07 — APPROVED** [review]
looks good

**Review by BeTheChange07 — APPROVED** [review]
looks good

**Review by BeTheChange07 — APPROVED** [review]

**Pull request merged**